acceptance/openstack/networking/v2/sgs/sg_test.go

```diff
@@ -31,17 +31,12 @@ func TestThrottlingSgs(t *testing.T) {
 	sg, err := secgroups.Create(clientCompute, createSGOpts).Extract()
 	th.AssertNoErr(t, err)
 
-	size := 20
+	size := 15
 	q := make(chan []string, size)
-	for i := 0; i < size*2; i++ {
-		go func() {
-			err := CreateMultipleSgsRules(clientNetworking, sg.ID, 400, q)
-			if err != nil {
-				return
-			}
-		}()
+	for i := 0; i < size; i++ {
+		go CreateMultipleSgsRules(clientNetworking, sg.ID, 47, i, q)
```

Comment:
acceptance/openstack/networking/v2/sgs/sg_test.go#L37

Error return value is not checked (errcheck)